core/src/test/scala/sttp/client3/testing/HttpTest.scala

```diff
@@ -409,6 +410,28 @@ trait HttpTest[F[_]]
         }
       }
 
+      if (supportsCustomMultipartEncoding) {
+        "send a multipart message containing body parts with explicitly set encodings" in {
+
+          val utf8Char = "\u00f6"
+          val replacementChar = "\ufffd"
```

Comment:
I think I would add a comment where this replacement char originates from (and maybe render both in comment, if they have some proper visualisations in utf-8?)

Reply:
corrected